test/e2e/adminUI/pages/fieldTypes/url.js

```diff
@@ -0,0 +1,39 @@
+var utils = require('../../../utils');
+
+module.exports = function UrlType(config) {
+	var self = {
+		selector: '.field-type-url[for="' + config.fieldName + '"]',
+		elements: {
+			label: '.FormLabel',
+			value: 'input[name="' + config.fieldName + '"]',
+		},
+		commands: [{
```

Comment:
Looks fine [here](https://github.com/jstockwin/keystone/blob/testing-e2e-urls/test/e2e/adminUI/pages/fieldTypes/url.js). You sure it isn't your window being too thin, and so is pushing line 15 onto two lines?